python-test.py

```diff
@@ -1,11 +1,11 @@
 print ("This is a tttsting code to check error catching")
 
-def simple_fn(range):
-    if range > 0:
```

Comment:
Improve function name `simple_fn` to be more descriptive, such as `print_range_with_seed`